test/integration/multikueue/suite_test.go

```diff
@@ -173,9 +151,37 @@ func managerAndMultiKueueSetup(mgr manager.Manager, ctx context.Context) {
 	err := multikueue.SetupIndexer(ctx, mgr.GetFieldIndexer(), managersConfigNamespace.Name)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
+	ctrlOpts := ctx.Value(framework.ManagerControllerOptsKey{}).([]multikueue.SetupOption)
 	err = multikueue.SetupControllers(mgr, managersConfigNamespace.Name,
-		multikueue.WithGCInterval(2*time.Second),
-		multikueue.WithWorkerLostTimeout(testingWorkerLostTimeout),
+		ctrlOpts...,
 	)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 }
+
+func multikueueBeforeAll(gcInterval time.Duration) {
```

Comment:
O dont like the name very much maybe we can make it `clustersSetup` or something.